### PR TITLE
feat(RiddleCommand): add riddle command

### PR DESCRIPTION
**Pull request by gautam-paranjape**
Small riddle command that fetches riddle answers, from an API.

### CODE REVIEW

**Review by Milo123459 — CHANGES_REQUESTED** [review]
Remove package-lock.json

**Comment by gautam-paranjape** [comment]
Fixed

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ config.json
 !.yarn/releases
 !.yarn/plugins
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+node_modules
```
Comment (Milo123459) [inline .gitignore:9]: why is node_modules being added? this project should never have a node_modules

Comment (gautam-paranjape) [reply .gitignore:9]: wait

Comment (gautam-paranjape) [reply .gitignore:9]: sorry it was because i was installing the dependencies locally, will fix

Comment (Milo123459) [reply .gitignore:9]: All good

File: src/commands/Fun Commands/RiddleCommand.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		client
+			.embed({ title: 'Riddle!' }, message)
+			.addField(response.question, `||${response.answer}||`)
+	);
+};
+
+export const name: string = 'riddle';
+export const category: string = 'fun';
+export const description: string =
+	'Fetch a random fact and answer from an API.';
```
Comment (Milo123459) [inline src/commands/Fun Commands/RiddleCommand.ts:19]: update description

Comment (gautam-paranjape) [reply src/commands/Fun Commands/RiddleCommand.ts:19]: linting did that let me fix

Comment (Milo123459) [reply src/commands/Fun Commands/RiddleCommand.ts:19]: Linting? I'm talking about the actual content of the description... Shouldn't that be changed?

Comment (gautam-paranjape) [reply src/commands/Fun Commands/RiddleCommand.ts]: Oh lmao ok

**Review by Milo123459 — APPROVED** [review]

**Pull request merged**